CHANGELOG.rst

```diff
@@ -1,6 +1,13 @@
 Changelog for Pulse3D
 =====================
 
+0.30.5 (unreleased)
+-------------------
+
+Changed:
```

Comment:
Also optical files were technically already supported, so we can remove the note about addition and put the remove of noise filtering, etc. under a `Fixed` section